### DISCUSSION TITLE
https://jsd.onmicrosoft.cn/npm/alist-web@3.17.0镜像失效了吗

### DISCUSSION THREAD

**Discussion by QiuYeDx** [AlistGo/alist#5054] [Q&A] [answered]
RT，请问 https://jsd.onmicrosoft.cn/npm/alist-web@3.17.0 这个镜像失效了吗？

今天突然发现js和css等静态文件全部404了，你们能获取到资源吗？

<img width="1241" alt="截屏2023-08-20 17 50 02" src="https://github.com/alist-org/alist/assets/71062076/3699a137-89c6-4ba8-803c-00e84bc0d364">

**Comment by xhofe** [comment, marked as answer]
你可以看这里 https://cdn.onmicrosoft.cn/ 它今天用完了 推荐使用其他的cdn

**Comment by anwen-anyi** [comment]
`onmicrosoft` 被人恶意刷了 两三千账单了，先关闭了。
